tests/test_ingest.py

```diff
@@ -120,8 +120,16 @@ def test_ingest_dense_matrix(self):
             "--file-id",
             "1234abc",
             "ingest_expression",
-            "--file-params",
-            '{"taxon_name": "Homo sapiens", "taxon_common_name": "human", "ncbi_taxid": "9606", "genome_assembly_accession": "GCA_000001405.15", "genome_annotation": "Ensemble 94"}',
+            "--taxon-name",
+            "Homo sapiens",
+            "--taxon-common-name",
+            "human",
+            "--ncbi-taxid",
+            "9606",
+            "--genome-assembly-accession",
+            "GCA_000001405.15",
+            "--genome-annotation",
+            "Ensemble 94",
```

Comment:
I'd suggest a global find-replace through all files to change "Ensemble 94" to "Ensembl 94".

```suggestion
            "Ensembl 94",
```